FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Core/Jobs/JobManager.cs

```diff
@@ -49,6 +50,9 @@ public async Task RunAsync(CancellationToken cancellationToken = default)
         {
             _logger.LogInformation("Job manager starts running.");
```

Comment:
We can just log $"Job manager starts running with version {}"

Reply:
Updated.